README.tfdoc.hcl

```diff
@@ -0,0 +1,338 @@
+header {
+  image = "https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg"
+  url   = "https://mineiros.io/?ref=terraform-google-service"
+
+  badge "build" {
+    image = "https://github.com/mineiros-io/terraform-google-service/workflows/Tests/badge.svg"
+    url   = "https://github.com/mineiros-io/terraform-google-service/actions"
+    text  = "Build Status"
+  }
+
+  badge "semver" {
+    image = "https://img.shields.io/github/v/tag/mineiros-io/terraform-google-service.svg?label=latest&sort=semver"
+    url   = "https://github.com/mineiros-io/terraform-google-service/releases"
+    text  = "GitHub tag (latest SemVer)"
+  }
+
+  badge "terraform" {
+    image = "https://img.shields.io/badge/Terraform-1.x-623CE4.svg?logo=terraform"
+    url   = "https://github.com/hashicorp/terraform/releases"
+    text  = "Terraform Version"
+  }
+
+  badge "tf-gcp-provider" {
+    image = "https://img.shields.io/badge/google-4-1A73E8.svg?logo=terraform"
```

Comment:
same here, the module is compatible with version 3 and 4, shall we somehow show this in the badge? can we maybe have something in terradoc, maybe something such as `provider_version` or so that we use to generate all badges and references? cc @mariux

Reply:
yes... i would actually love to drop 3.x support asap.. wdyt?

Reply:
in this case we can leave 3. support undocumented.